### DISCUSSION TITLE
Username in jwt access token

### DISCUSSION THREAD

**Discussion by mrdcb** [zitadel/zitadel#6350] [Q&A] [answered]
Is there a list of claims I can include in in a jwt access token?
I'm able to include metadata and roles in the token. I also get the user_id via the sub claim.
But it seems it is not possible to include more userinfo? Even when I include a profile scope…
(A human readable name would be nice for logging)

Is there a difference between a human user and a service user?

**Comment by mffap** [comment, marked as answer]
> Is there a list of claims I can include in in a jwt access token? 

https://zitadel.com/docs/apis/openidoauth/scopes
https://zitadel.com/docs/apis/openidoauth/claims

also you can do custom claims

see examples: https://github.com/zitadel/actions/tree/main/examples

> I'm able to include metadata and roles in the token. I also get the user_id via the sub claim. But it seems it is not possible to include more userinfo? Even when I include a profile scope… (A human readable name would be nice for logging)

Can you check if it works when you enable "User Info inside ID Token"?
https://zitadel.com/docs/guides/manage/customize/user-metadata#send-metadata-inside-the-id-token

> Is there a difference between a human user and a service user?

https://zitadel.com/docs/concepts/structure/users

**Comment by mrdcb** [comment]
Thank you for your quick response :-)

I'll look into custom claims which looks nice.

So according to https://zitadel.com/docs/apis/openidoauth/claims I can not include a `name` in an Access Token (When JWT). 
Is the first table the complete list of claims I can add to an Access Token (JWT)? What about the other tables?
I'm not always sure which scope will get me what claim in the end.

> Can you check if it works when you enable "User Info inside ID Token"?
In my app I have this enabled (And the id token has all the info I need) But I want to use JWT as Access Tokens. And even though it is an Access Token in my case adding some userinfo would simply things…
